helmfile/charts/prometheus-alerts/templates/alerts/packets-dropped.yaml

```
@@ -0,0 +1,60 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.networkpolicies }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "packets-dropped" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+    {{- include "prometheus-alerts.labels" . | nindent 4 }}
+    {{- if .Values.defaultRules.alertLabels }}
+    {{- toYaml .Values.defaultRules.alertLabels | nindent 4 }}
+    {{- end }}
+  {{- if .Values.defaultRules.annotations }}
+  annotations:
+    {{- toYaml .Values.defaultRules.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  groups:
+  - name: packets-dropped
+    rules:
+    - alert: FrequentPacketsDroppedToWorkload
+      annotations:
+        description: Number of packets dropped to a workload because no policies matched them
+        summary: Number of packets dropped to a workload
+      expr: |
+        increase(no_policy_drop_counter{type="tw", cluster=~".*", pod_namespace=~".*", exported_pod=~".*"}[1m]) > 0
+      for: 5m
+      labels:
+        severity: info
+    - alert: ScarcePacketsDroppedToWorkload
+      annotations:
+        description: Number of packets dropped to a workload because no policies matched them
+        summary: Number of packets dropped to a workload
+      expr: |
+        increase(no_policy_drop_counter{type="tw", cluster=~".*", pod_namespace=~".*", exported_pod=~".*"}[3h]) > 20
+      for: 6h
+      labels:
+        severity: info
+    - alert: FrequentPacketsDroppedFromWorkload
+      annotations:
+        description: Number of packets dropped from a workload because no policies matched them
+        summary: Number of packets dropped from a workload
+      expr: |
+        increase(no_policy_drop_counter{type="fw", cluster=~".*", pod_namespace=~".*", exported_pod=~".*"}[1m]) > 0
+      for: 5m
+      labels:
+        severity: info
+    - alert: ScarcePacketsDroppedFromWorkload
+      annotations:
+        description: Number of packets dropped from a workload because no policies matched them
+        summary: Number of packets dropped from a workload
+      expr: |
+        increase(no_policy_drop_counter{type="fw", cluster=~".*", pod_namespace=~".*", exported_pod=~".*"}[3h]) > 20
+      for: 6h
+      labels:
+        severity: info
+
+{{- if .Values.defaultRules.additionalRuleLabels }}
+{{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
+{{- end }}
```

I think this should be for each rule since it's the global extra rule label thing:

```suggestion
      labels:
        severity: info
{{- if .Values.defaultRules.additionalRuleLabels }}
{{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
{{- end }}
```

Like this one https://github.com/elastisys/compliantkubernetes-apps/blob/aeddafali/alert-for-blocked-network-traffic/helmfile/charts/prometheus-alerts/templates/alerts/kubernetes-resources.yaml#L30-L65

---

Done, thanks a lot 👍🏽